gef.py

```diff
@@ -8687,34 +8687,32 @@ def _syntax_(self):
     def __init__ (self):
         super(GenericFunction, self).__init__(self._function_)
 
-    def invoke(self, offset=gdb.Value(0)):
+    def invoke(self, arg=gdb.Value(0)):
```

Comment:
I don't think a default value makes sense here either. We can use `*args` to make this variadic and if the arg is missing use `gdb.Value(0)`.

Again, I don't think that it makes sense for the generic function to handle specific types (unless we wanted to add expression parsing here).